R/dataframe-frame.R

```diff
@@ -539,3 +539,26 @@ dataframe__rechunk <- function() {
     self$`_df`$rechunk()
   })
 }
+
+#' @inherit lazyframe__unnest title description params
+#' @inherit as_polars_df return
+#' @examples
+#' df <- pl$DataFrame(
+#'   a = 1:5,
+#'   b = c("one", "two", "three", "four", "five"),
+#'   c = 6:10
+#' )$
+#'   select(
+#'   pl$struct("b"),
+#'   a_and_c = pl$struct(c("a", "c"))
+#' )
+#' df
+#'
+#' df$unnest("a_and_c")
+dataframe__unnest <- function(...) {
+  wrap({
+    columns <- unlist(list2(...))
```

Comment:
`unlist()` is dangerous.
Please use this as like `pl__col()`

https://github.com/eitsupi/neo-r-polars/blob/473ade21c048bd4a80cec7b201dfb978f5c5b5c1/R/utils-check.R#L34-L62

Reply:
Should be good now, but note that there's a different behavior between eager and lazy: https://github.com/pola-rs/polars/issues/20128

Reply:
Ok, please remove the dataframe method and use the lazyframe one.